exchanges/exchange.go

```diff
@@ -1876,3 +1874,67 @@ func (b *Base) ParallelChanOp(channels []subscription.Subscription, m func([]sub
 
 	return errs
 }
+
+// Bootstrap function allows for exchange authors to supplement or override common startup actions
+// If exchange.Bootstrap returns false or error it will not perfom any other actions.
+// If it returns true, or is not implemented by the exchange, it will:
+// * Print debug startup information
+// * UpdateOrderExecutionLimits
+// * UpdateTradablePairs
+func Bootstrap(ctx context.Context, b IBotExchange) error {
+	if continueBootstrap, err := b.Bootstrap(ctx); !continueBootstrap || err != nil {
+		return err
+	}
+
+	if b.IsVerbose() {
+		wsURL := ""
+		wsEnabled := false
+		if w, err := b.GetWebsocket(); err != nil {
+			wsURL = w.GetWebsocketURL()
+			wsEnabled = w.IsEnabled()
+		}
+		log.Debugf(log.ExchangeSys, "%s Websocket: %s. (url: %s).\n", b.GetName(), common.IsEnabled(wsEnabled), wsURL)
```

Comment:
Not all exchanges support websocket so we can check to see if the exchange supports it and only print the above if it does so

Reply:
It's also wrong AF: `[DEBUG] | EXCHANGE | 30/01/2024 12:13:57 | Bybit Websocket: Disabled. (url: ).`
🤷 
On it.

Reply:
Yeah. That was stupid. Should have been `err == nil`.

Hope it's okay, but I've decided to be specific that the websocket is unsupported. We're being verbose, after all.

Fixed 3cccf3c0